### PR TITLE
Set https_proxy env var for covscan image builds

### PR DESCRIPTION
**Pull request by locriandev**
https://coreos.slack.com/archives/GDBRP5YJH/p1655720187867989

### CODE REVIEW

File: doozerlib/coverity.py [hunk 1]
```diff
@@ -273,6 +273,9 @@ def _covscan_prepare_parent(cc: CoverityContext, parent_image_name, parent_tag)
 LABEL DOOZER_COVSCAN_GROUP_PARENT={cc.runtime.group_config.name}
 USER 0
 
+# Set https proxy
+ENV https_proxy http://proxy.util.phx1.redhat.com:8080
```
Comment (joepvd) [inline doozerlib/coverity.py:277]: Prefer not to hardcode a string in this location. Maybe query `os.environ` for the existence of `(https?|no)_proxy` vars and set them if defined?

Comment (vfreex) [reply doozerlib/coverity.py:277]: We may change to another proxy over time. Agree this shouldn't been hardcoded.

An option could be using `ARG` directive in Dockerfile then pass proxy related environment variables to the podman command line.

Comment (vfreex) [reply doozerlib/coverity.py:277]: We may change to another proxy over time. Agree this shouldn't been hardcoded.

An option could be using `ARG` directive in Dockerfile then pass proxy related environment variables to the podman command line.

**Review by vfreex — CHANGES_REQUESTED** [review]

**Comment by locriandev** [comment]
If merged, will require https://github.com/openshift/aos-cd-jobs/pull/3113

**Review by vfreex — APPROVED** [review]
/lgtm

**Pull request merged**